bitcoin/src/network/constants.rs

```diff
@@ -108,6 +112,40 @@ impl Network {
     }
 }
 
+impl fmt::Display for Network {
+    fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
+        use Network::*;
+
+        let s = match *self {
+            Bitcoin => "bitcoin",
+            Testnet => "testnet",
+            Signet => "signet",
+            Regtest => "regtest",
+        };
+        f.pad(s)
```

Comment:
I'm afraid pad will cut off letters if someone specifies too low precision.

Reply:
So we should have thought harder and used `write!()` like normal, right?

Reply:
For now, yes. We will need some helper fn to format non-integers without losing precision.